Source/automap.cpp

```diff
@@ -830,6 +1357,107 @@ void InitAutomap()
 {
 	size_t tileCount = 0;
 	std::unique_ptr<AutomapTile[]> tileTypes = LoadAutomapData(tileCount);
+
+	if (IsAnyOf(leveltype, DTYPE_CATACOMBS)) {
+		tileTypes[41] = { AutomapTile::Types::FenceHorizontal };
+	}
+	if (IsAnyOf(leveltype, DTYPE_TOWN, DTYPE_CAVES, DTYPE_NEST)) {

```

Comment:
This should all be `else if`. Probably a `switch` is a better choice here actually.